observations/README.md

```diff
@@ -0,0 +1,1279 @@
+Blood Pressure Observations are mapped to a `CompoundStatement` with `ObservationStatement` components for the systolic 
+and diastolic readings and a optional `NarrativeStatement` for additional information.
+
+### Parent Compound Statement
+
+| Mapped to (XML HL7 CompoundStatement) | Mapped from (JSON FHIR / other source )                                                                                                                      |
+|---------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| id                                    | Unique ID generated by the adaptor                                                                                                                           |
+| code \[@code]                         | `Observation.code.coding[0].code`                                                                                                                            |
+| code \[@displayName]                  | `Observation.code.coding[0].display`                                                                                                                         |
+| code \[@codeSystem]                   | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15"`                                                                                                             |
+| code / originalText                   | `coding.text` <br/> or else `coding.display` <br/> or else `coding.extension[index].value` where `coding.extension[index].url` equals `"descriptionDisplay"` |
+| statusCode                            | fixed value = `"COMPLETE"`                                                                                                                                   |
+| effectiveTime                         | `Observation.effectiveDateTime` or else `Observation.effectivePeriod`                                                                                        |
+| availabilityTime \[@value]            | `Observation.effectiveDateTime` or else `Observation.effectivePeriod.start`                                                                                  |
+| Participant \[@typecode]              | fixed value = `"PRF"`                                                                                                                                        |
+| Participant / agentRef / id \[@root]  | `Observation.performer[0]`                                                                                                                                   | 
+
+### Observation components
+
+The blood pressure should have two `ObservationStatement` components, one with a systolic and one with a diastolic 
+SNOMED code.  
```

Comment:
I wonder if we could be clearer about the `\[index]` being used below is a value based on identifying the systolic and diastolic components.

Reply:
done